### DISCUSSION TITLE
Quotation mark in rtsp-simple-server.yml (encryption: "no")

### DISCUSSION THREAD

**Discussion by MrGrosNounours** [bluenviron/mediamtx#632] [Questions]
Hello,

Since the last update, the "rtsp-simple-server.yml" config file have quotation mark for the line encryption (encryption: "no").
Is it normal ?

PS: This project is awsome.

**Comment by aler9** [comment]
Hello, the change was needed since this is a boolean value and means "false":

```yml
encryption: no
```

while this is a string value and means "no":

```yml
encryption: "no"
```

In order to align the configuration schema with the API schema, the encryption field now is a string and can be decoded/encoded from/to strings only.

**Reply by MrGrosNounours** [reply]
Thanks for your reply.

[a comment by an automated account is not shown]
